### PR TITLE
Doco

### PR DESCRIPTION
**Pull request by drmcghee**
Added to existing documentation and a scripts directory for the ngrok file/run

### CODE REVIEW

File: scripts/ngrok.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+authtoken: YOUR_AUTH_TOKEN
```
Comment (ankitbko) [inline scripts/ngrok.yml:1]: We already have ngrik.template.yml, do we need this as well?

File: src/appsettings.template.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (ankitbko) [inline src/appsettings.template.json]: Do we need this. The `appsettings.json` as exact same values

File: .gitignore [hunk 3]
```diff
@@ -362,4 +363,6 @@ MigrationBackup/
 # Fody - auto-generated XML schema
 FodyWeavers.xsd
 
-cache/
\ No newline at end of file
+cache/
+
+appsettings.json
```
Comment (ankitbko) [inline .gitignore]: appsettings can be committed to the code. Its part of .net core project structure

File: scripts/ngrok.Designer.cs [hunk 4]
```diff
@@ -0,0 +1,242 @@
+﻿//------------------------------------------------------------------------------
```
Comment (ankitbko) [inline scripts/ngrok.Designer.cs]: Do we need this file?

**Review by ankitbko — CHANGES_REQUESTED** [review]

**Comment by deesebast** [comment]
Suggestion:  Can we rename debug.md to something like 'Development guide'.  In the doc, we could have two sections, one each for development and deployment.

**Comment by ankitbko** [comment]
> Suggestion: Can we rename debug.md to something like 'Development guide'. In the doc, we could have two sections, one each for development and deployment.

Yup

**Comment by roshni-poddar** [comment]
@drmcghee could you please resolve this?

**Pull request merged**